resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/SparkKubernetesClientFactory.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.deploy.k8s
+
+import java.io.File
+
+import com.google.common.base.Charsets
+import com.google.common.io.Files
+import io.fabric8.kubernetes.client.{Config, ConfigBuilder, DefaultKubernetesClient, KubernetesClient}
```

Comment:
I don't think Scalastyle checks for line lengths in imports. But these imports for the Fabric8 client have typically been quite long. I've been wondering if it's worth just importing the whole package with `io.fabric8.kubernetes.client._`.

Reply:
It could also be split into multiple lines, sorted alphabetically ...

Reply:
It could be, but I don't quite see that elsewhere in the project? Do we mean something like this:

```
import io.fabric8.kubernetes.client.{
  Config,
  ConfigBuilder,
  ...
}
```

or do we mean something more like this:

```
import io.fabric8.kubernetes.client.Config
import io.fabric8.kubernetes.client.ConfigBuilder
...
```

Either case I don't think is necessarily standard in the project, but I have seen a few places where the entire package is imported via the `_` token.

Reply:
@jiangxb1987 Is this the convention across the codebase? I don't think most IDEs will preserve that formatting.